### PR TITLE
16 t watch add support for greek letters or any other non latin

### LINKED ISSUE #16: (T-Watch) Add support for Greek letters (or any other non-latin)

### PR DESCRIPTION
**Pull request by themelispan**

### CODE REVIEW

File: Arduino/TWatch 2.0/DisplayTime.ino [hunk 1]
```diff
@@ -89,7 +89,20 @@ void displayTime(boolean fullUpdate) {
...
+    u8f.drawGlyph( 0, 230, 0x0e200);     // Power Supply
+    u8f.drawGlyph(12, 230, 0x0e201);     // Charging
+    u8f.drawGlyph(24, 230, 0x0e10a);     // Right Arrow
+    u8f.drawGlyph(36, 230, 0x0e24b);     // full Battery*/
+
+    //a test for fonts
+    /*u8f.setFont( u8g2_font_unifont_t_greek);     // greek font
+    u8f.setFontMode(1);                 // use u8g2 transparent mode (this is default)
+    u8f.setCursor(100, 230);               // start writing at this position
+    u8f.print("Δοκιμή it's a test");                 // a string in greek to test it*/
```
Comment (themelisx) [inline Arduino/TWatch 2.0/DisplayTime.ino]: Please remove the comments before commit

File: Arduino/TWatch 2.0/TWatch 2.0.ino [hunk 2]
```diff
@@ -144,7 +146,14 @@ void setup() {
...
   ttgo->tft->fillScreen(TFT_BLACK);
-  ttgo->tft->setTextColor(TFT_WHITE, TFT_BLACK);  // Note: the new fonts do not draw the background colour
+  ttgo->tft->setTextColor(TFT_WHITE);  // Note: the new fonts do not draw the background colour
+  
+  // Turn on the backlight
+  ttgo->openBL();
+  tft = ttgo->tft;
+  u8f.begin(*tft);
+  u8f.setForegroundColor(TFT_WHITE);
+
```
Comment (themelisx) [inline Arduino/TWatch 2.0/TWatch 2.0.ino:158]: Please add some comments about your changes.
It doesn't seem to relevant with the issue description

File: Arduino/TWatch 2.0/TWatch 2.0.ino [hunk 3]
```diff
@@ -177,6 +186,8 @@ void setup() {
   mmonth = tnow.month;
   yyear = tnow.year;
 
+  displayTime(true);
+
```
Comment (themelisx) [inline Arduino/TWatch 2.0/TWatch 2.0.ino:192]: The same as above

**Review by themelisx — CHANGES_REQUESTED** [review]

File: Arduino/TWatch 2.0/TWatch 2.0.ino (hunk 2 above)
Comment (themelispan) [reply Arduino/TWatch 2.0/TWatch 2.0.ino:158]: done

**Review by themelisx — APPROVED** [review]

**Pull request merged**